### PR TITLE
Patch3

### PR DESCRIPTION
**Pull request by bmadhavi74**
Updated the rule by adding the sensor, fields, variables and trigger conditions.

### CODE REVIEW

File: community_supplied/Interface/oc-interface-throughput.rule [hunk 1]
```diff
@@ -1,11 +1,350 @@
...
+                    juniper {
+                        operating-system junos {
+                            products EX {
+                                releases 17.3R1 {
+                                    release-support min-supported-release;
+                                    platform all;
+                                }
+                            }
+                            products MX {
+                                releases 15.1R1 {
```
Comment (gvarao) [inline community_supplied/Interface/oc-interface-throughput.rule:339]: minimum supported version is. 16.2R1.

File: community_supplied/Interface/oc-interface-throughput.rule [hunk 2]
```diff
@@ -1,11 +1,350 @@
...
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 3.1.0;
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products EX {
+                                releases 17.3R1 {
```
Comment (gvarao) [inline community_supplied/Interface/oc-interface-throughput.rule:333]: I don't think EX. supports OC

File: community_supplied/Interface/oc-interface-throughput.rule [hunk 3]
```diff
@@ -1,11 +1,350 @@
...
+                            message "$interface-name output bandwidth utilization ($out-bw-util %) is normal. Mbps:$out-mbps Output bytes:$output-bytes";
+                        }
+                    }
+                }
+            }
+            /*
+            * Variables with default values. Default values can be changed
+            * while deploying playbook instance.
+            */
+            variable in-stats-name {
```
Comment (gvarao) [inline community_supplied/Interface/oc-interface-throughput.rule:300]: No need of templating in, out octets as it is bw rule hence remove in-octets and out-octets variable.

File: community_supplied/Interface/oc-interface-throughput.rule (hunk 2 above)
Comment (bmadhavi74) [reply community_supplied/Interface/oc-interface-throughput.rule:306]: EX supports a particular platform in OC, so i made the required changes

**Pull request merged**